### PR TITLE
Style search section

### PR DESCRIPTION
**Pull request by alternadiva**
I've styled the first part of the website according to your design ideas, still a  long way to go, though. Let me know your thoughts.
desktop

mobile

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,21 +7,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Home Traveller</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quattrocento+Sans:wght@400;700&display=swap" rel="stylesheet">
 </head>
 
 <body>
     <header>
         <h1>Home Traveller</h1>
+        <img src="./assets/home_traveller.png" alt="person's watching Italian movie and eating pizza" id="index-img">
```
Comment (sonianb) [inline index.html:16]: I absolutely adore this photo! Did you use canva to make it?

File: style.css [hunk 2]
```diff
@@ -1,3 +1,100 @@
...
+  padding: 0;
+}
+
+#search-section {
+  max-width: 800px;
+  min-width: 300px;
+  height: auto;
+  min-height: 150px;
+  padding-top: ;
+  background-color: rgb(185, 221, 198);
```
Comment (sonianb) [inline style.css:44]: Great color choice! I would put the colors in a variable, so we could reuse them in the project.

**Review by sonianb — APPROVED** [review]
Love the color choice and the overall design!

File: index.html (hunk 1 above)
Comment (alternadiva) [reply index.html:16]: Yes, montaged different graphics together 😄

File: style.css (hunk 2 above)
Comment (alternadiva) [reply style.css:44]: Good idea!

**Pull request merged**